astropy/utils/masked/tests/test_table.py

```diff
@@ -0,0 +1,111 @@
+# coding: utf-8
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import numpy as np
+from numpy.testing import assert_array_equal
+import pytest
+
+from astropy import units as u
+from astropy.table import QTable
+
+from astropy.utils.masked import Masked
+
+from .test_masked import assert_masked_equal
+
+try:
```

Comment:
When rebase, can use the new centralized flags.  #11490

Reply:
OK, done